### PR TITLE
Fixed display bug for job failures. Fixed status to match latest vers…

### PR DESCRIPTION
**Pull request by thomasmaurel**
…ion from DBA. Re-instated the reset failed job button but only for superuser, the button now update the job status to 'Manually Launched by Production team' which will make sure that DBAs rundeckwill restart the job

### CODE REVIEW

File: ensembl_dbcopy/models.py [hunk 1]
```diff
@@ -160,8 +165,10 @@ def table_status(self):
                 return 'Failed'
             elif self.job_id.status == 'Creating Requests':
                 return 'Submitted'
-            else:
+            elif self.job_id.status == 'Processing Requests':
                 return 'Running'
+            else:
+                return 'Submitted'
         else:
             return 'Submitted'
```
Comment (radome) [inline ensembl_dbcopy/models.py:173]: Similarly here, can we simplify with only one `return 'Submitted'`?

File: ensembl_dbcopy/models.py [hunk 2]
```diff
@@ -87,6 +87,8 @@ def overall_status(self):
                 return 'Running'
             elif self.transfer_logs.count() > 0 and self.status=='Creating Requests':
                 return 'Submitted'
+            else:
+                return 'Submitted'
         else:
             return 'Submitted'
```
[the remark this answers is not in the document]
Comment (thomasmaurel) [reply ensembl_dbcopy/models.py]: Makes perfect sense thanks!

File: ensembl_dbcopy/models.py (hunk 1 above)
Comment (thomasmaurel) [reply ensembl_dbcopy/models.py]: Makes perfect sense thanks!

File: ensembl_dbcopy/models.py [hunk 3]
```diff
@@ -111,6 +113,9 @@ def detailed_status(self):
                 if self.status=='Creating Requests':
                     return {'status_msg': 'Submitted', 'table_copied': table_copied, 'total_tables': total_tables,
                             'progress': progress}
+                else:
+                    return {'status_msg': 'Submitted', 'table_copied': table_copied, 'total_tables': total_tables,
+                        'progress': progress}
             else:
                 return {'status_msg': 'Submitted', 'table_copied': table_copied, 'total_tables': total_tables,
                         'progress': progress}
```
[the remark this answers is not in the document]
Comment (thomasmaurel) [reply ensembl_dbcopy/models.py]: Makes perfect sense thanks!

**Pull request merged**